runtime/src/blueoil_image.cpp

```diff
@@ -100,26 +100,39 @@ Tensor ResizeHorizontal_BiLinear(const Tensor &tensor, const int width) {
   const int channels  = shape[2];
   const int height = srcHeight;
   Tensor dstTensor({height, width, channels});
-  float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
-  int xSrcWindow = std::floor(1/xScale);
-  xSrcWindow = (xSrcWindow < 2)? 2 :xSrcWindow;
+  const float xScale = static_cast<float>(width) / static_cast<float>(srcWidth);
+  const float xSrcWindow = (xScale < 1.0)?(1.0f / xScale):1.0;
```

Comment:
Could you insert a space before/after `?` and `:`?

Reply:
https://github.com/blue-oil/blueoil/pull/611/commits/5dd6c9aee43af40bfbf32b738d2b0ffb07631947#diff-37230609db7212ca72dd52076761c190L104